lib/ui/screens/about.dart

```diff
@@ -0,0 +1,271 @@
+
+  @override
+  void initState() {
+    super.initState();
+    // Apparently there is no pretty print platform name, so we just set the exceptions here
+    if (Platform.isIOS)
+      os = "iOS";
+    else if (Platform.isMacOS)
+      os = "macOS";
+    initPlatformState();
+  }
+
+  initPlatformState() async {
+    PackageInfo packageInfo = await PackageInfo.fromPlatform();
+    String appName = packageInfo.appName;
+    String appVersion = packageInfo.version;
+
+    if (!mounted) return;
+
+    setState(() {
+      _appName = appName;
+      _appVersion = appVersion;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return CobbleScaffold.tab(
+      title: " ", // We want an empty appbar
+      bottom: PreferredSize(
+        preferredSize: const Size.fromHeight(156.0),
+        child: Container(
+            height: 156.0,
+            child: Padding(
+              padding: EdgeInsets.fromLTRB(16.0, 14.0, 16.0, 9.0),
+              child: Column(children: <Widget>[
+                Row(children: <Widget>[
+                  SizedBox(
+                    width: 64.0,
+                    height: 64.0,
+                    child: Image(image: Svg('images/app_icon.svg')),
+                  ),
+                  SizedBox(width: 20.0),
+                  Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          _appName,
+                          style: Theme.of(context).textTheme.headline5,
+                        ),
+                        SizedBox(height: 4.0),
+                        Text(
+                          tr.aboutPage.versionString(
+                              version: _appVersion, platform: os),
+                          style: Theme.of(context).textTheme.bodyText2,
+                        ),
+                      ]),
+                ]),
+                SizedBox(height: 14.0),
+                Row(children: <Widget>[
+                  Expanded(
+                      child: SizedBox(
+                          height: 40.0,
+                          child: RaisedButton(
+                            child: Row(children: <Widget>[
+                              Icon(RebbleIcons.rocket, size: 25),
+                              SizedBox(width: 8),
+                              Text(tr.aboutPage.sourceCode.toUpperCase()),
+                            ]),
+                            onPressed: () => {launchURL(sourceURL)},
+                          ))),
+                  SizedBox(width: 16),
+                  Expanded(
+                      child: SizedBox(
+                          height: 40.0,
+                          child: CobbleButton(
+                            outlined: false,
+                            label: tr.aboutPage.licenses,
+                            onPressed: () => {showLicensePage(context: context)},
```

Comment:
Lambdas in Dart have different syntax. In this case you created new Map with a single key-value pair instead of simply calling `showLicensePage()` 😉

Reply:
As far as I can tell, this works perfectly

Reply:
It might work correctly but it's not doing what you think it does. 
```
() => {showLicensePage(context: context)},
```
is same as
```
() {
  final value = showLicensePage(context: context);
  final key = value.toString();
  return { key: value };
}
```